# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../altermime/makefile-destdir.patch
# Copyright (C) 2013 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Description: Adds ${DESTDIR} variable to Makefile
Author: Julien Valroff <julien@debian.org>
Forwarded: not-needed

--- a/Makefile
+++ b/Makefile
@@ -25,11 +25,11 @@
 # Build Install
 install: altermime
 	strip altermime
-	cp altermime /usr/local/bin
-	chmod a+rx /usr/local/bin/altermime
+	cp altermime ${DESTDIR}
+	chmod a+rx ${DESTDIR}/altermime
 
 uninstall:
-	rm -f /usr/local/bin/altermime
+	rm -f ${DESTDIR}/altermime
 
 clean:
 	rm -f *.o altermime
